csrc/device_lower/utils.cpp

```diff
@@ -769,8 +769,20 @@ AllocPosInfo getAllocPosInfo(
 
   bool outer_alloc_found = false;
 
+  int64_t stop_position = tv->getComputeAtPosition();
```

Comment:
Use the `stage_slice_position` instead of `compute_at_position` to allocate memory for TensorView.

Reply:
This change sounds makes sense, and seems to be independent to the loop mapping change. Could you post a separate PR for it?

Reply:
It is relatively stand-alone, but you can't make a complex test without updating IdModel in some way.

Reply:
#4550 changed the allocation behavior to account for `stage_slice_position`